### PR TITLE
Feat: Create Schema Based on Diagram

### PR DESCRIPTION
**Pull request by leehaney254**
- Created a file named schema_based_on_diagram.sql where we implement the database from the diagram
- Created a Join table from the many-to-many relationships
-  Created FK indexes

### CODE REVIEW

File: schema_based_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+   FOREIGN KEY(medical_histories_id) REFERENCES medical_histories(id),
+  CONSTRAINT fk_treatments
+   FOREIGN KEY(treatments_id) REFERENCES treatments_id
+);
+
+-- Create a foreign key index
+CREATE INDEX patients_id_idx ON Medical_histories(patients_id);
+CREATE INDEX medical_history_id_idx ON invoices(medical_history_id);
+CREATE INDEX invoice_id_idx ON invoice_items(invoice_id);
+CREATE INDEX treatment_id_idx ON invoice_items(treatment_id);
```
Comment (Munsa1) [inline schema_based_diagram.sql]: - [ ] Kindly add indexes for all foreign keys, this includes the foreign keys for the join table as well. as this is a requirement

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @leehaney254 @houdamzari, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Created all tables in the provided schema :heavy_check_mark: 
Added all required foreign keys and join table to enforce relationships :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_diagram.sql (hunk 1 above)
Comment (leehaney254) [reply schema_based_diagram.sql]: Hello! Thank you. I have added indexes to the join table. 👍

**Review by julie-ify — APPROVED** [review]
Hi @leehaney254 and @houdamzari,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**